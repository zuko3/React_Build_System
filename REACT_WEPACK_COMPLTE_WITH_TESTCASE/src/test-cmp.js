import React, { Component } from "react";
import "./style/style.css";
import imageurl from "./images/logo.png";

export const TestCmp = props => (
  <h3>
    <img
      src={imageurl}
      style={{
        height: 40,
        width: 40
      }}
    />
    <p> I am th test component</p>
  </h3>
);
