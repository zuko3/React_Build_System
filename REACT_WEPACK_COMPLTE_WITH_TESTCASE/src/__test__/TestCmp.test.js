import React from "react";
import { TestCmp } from "../test-cmp";
import { shallow } from "enzyme";

describe("should render properly", () => {
  it("shuld check snapshot", () => {
    const wrapper = shallow(<TestCmp />);
    expect(wrapper).toMatchSnapshot();
  });
});
