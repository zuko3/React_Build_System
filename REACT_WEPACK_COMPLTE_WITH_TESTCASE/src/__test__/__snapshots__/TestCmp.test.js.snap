// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render properly shuld check snapshot 1`] = `
<h3>
  <img
    src={null}
    style={
      Object {
        "height": 40,
        "width": 40,
      }
    }
  />
  <p>
     I am th test component
  </p>
</h3>
`;
